subprojects/packagefiles/flac/src/libFLAC/meson.build

```diff
@@ -4,10 +4,25 @@ else
   extra_args = ['-DFLAC__NO_DLL']
 endif
 
+avx2_args = []
+
 if cc.get_argument_syntax() == 'msvc' and cdata.get('WITH_AVX')
-  extra_args += '/arch:AVX2'
+  avx2_args += '/arch:AVX2'
```

Comment:
hrm so this is MSVC only. Does AVX2 support work witth GCC/Clang?

Reply:
It employs [function multiversioning](https://gcc.gnu.org/onlinedocs/gcc-9.1.0/gcc/Function-Multiversioning.html) for non-MSVC: https://github.com/xiph/flac/blob/72787c3fee472809fb3f819558c7176066f0ee44/src/libFLAC/include/private/cpu.h#L96

Reply:
at runtime. interesting.